config/config_mac_gnu.sh

```diff
@@ -24,20 +21,15 @@ export DOWNLOAD_ONLY=N
 export STACK_EXIT_ON_FAIL=Y
 export WGET="wget -nv --no-check-certificate "
 
-#
 # gfortran-10 needs the following
-export STACK_esmf_FFLAGS="-fallow-argument-mismatch -fallow-invalid-boz"
-export STACK_madis_FFLAGS="-fallow-argument-mismatch -fallow-invalid-boz"
-export STACK_mapl_FFLAGS="-fallow-argument-mismatch -fallow-invalid-boz"
-#
-# User specific aliases and functions
-export PATH=/usr/local/bin/:$PATH
-export LIBDIRS=/usr/local/lib
-#
+export STACK_FFLAGS="-fallow-argument-mismatch -fallow-invalid-boz -march=native"
```

Comment:
I'm fine with this `-march=native`, but I don't understand why this is needed. I don't see why Proj attempts to use an instruction that it can't use.

Reply:
The flag "-march=native" is needed for CXXFLAGS only. Cleaned up other flags, FFLAGS, FCFLAGS